src/shims/posix/thread.rs

```diff
@@ -99,7 +99,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         let this = self.eval_context_mut();
         this.assert_target_os("linux", "prctl");
 
-        if args.len() < 1 {
+        if args.is_empty() {
```

Comment:
I'd prefer to keep `< 1` for consistency with all the other checks (here and in other vararg shims) for the number of arguments.